roles/self_hosted_first_host/defaults/main.yml

```diff
@@ -27,6 +27,8 @@ config_dir: /etc/ansible-ovirt
 host_id: 1
 host_name: hosted_engine_{{ host_id }}
 compatibility_version: 4.0
+# product stand for decide to install ovirt/rhv appliance
+product: rhv
```

Comment:
Maybe it will be better to set the default to ovirt.

I don't really understand the connection between OS to product.
We basically working with RHEL and sometimes wants to install ovirt from upstream
So the condition:
'{{ package_name **if ansible_distribution == "RedHat"** else "ovirt-engine-appliance" }}'
it's problematic for us.
wdyt?